README.md

```diff
@@ -192,40 +179,7 @@ Example command:
 
 ### Usage
 
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
-
-### Deployment
-
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
-
-```
- -->
-
+To use queries provided in this project run psql command and copy all commands one by one in psql shell
```

Comment:
- [ ] You have submitted a project, that does not have a professional Readme file make sure no text description from the template is present, all sections are required except the FAQ section 

![Screenshot](https://github.com/elad237/Vet-clinic-database/assets/79058364/9d6adcae-50ad-4129-976a-33a2739e0629)


- [ ] Kindly remove  React.js in the Client Tech Stack and Express.js in the Server Tech Stack since we're not using them in the project 👇🏻 

![Screenshot1](https://github.com/elad237/Vet-clinic-database/assets/79058364/e2c41f54-ad49-4718-8ef6-ebc752352dbf)


- [ ] I also recommend that you delete the Note: 👇🏻 in your License section as these are instructions on how to complete the License section.


![image](https://user-images.githubusercontent.com/79058364/220769702-9dbcadda-aa00-4d1c-990f-cf6ad4f5c6c9.png)